packages/shared/src/components/utilities/common.tsx

```diff
@@ -230,3 +230,6 @@ export const formatReadTime = (minutes: number): string => {
 };
 
 export const lazyCommentThreshold = 5;
+
+export const singleLineClamp =
```

Comment:
Not sure if @omBratteng introduced some generic already in his PR?

Reply:
Let me know if I should update anything.

Reply:
Do you mean the [PR about overflow](https://github.com/dailydotdev/apps/pull/2735/files)? Doesn't seem like it introduces a common CSS for this.

Reply:
It was this one @sshanzel 
https://github.com/dailydotdev/apps/blob/687b1b4ef11c45480c49c2892fb8001f96eb2a8f/packages/shared/src/components/utilities/common.tsx#L62

Reply:
I created `TruncateText` which has the `max-w-full shrink truncate` classes, but that's for the "child" element. What needs to be applied to the parent often element varies

Reply:
Ah, nice. It seems the `truncate` has all the properties needed.

Reply:
Yeah, and if it's inside of a flex, using the `TruncateText` will give you the other classes you might need (again, depends on the implementation <img src="https://emoji.slack-edge.com/T0141MGU324/hidethepain/20b6dfd560efad16.jpg" height="24" />

Reply:
Sometimes that one works alone, sometimes you need the others, and sometimes you need some extra in the parent.
<img src="https://emojis.slackmojis.com/emojis/images/1680519815/65011/old-man-yells-at-css.png"/><img src="https://emojis.slackmojis.com/emojis/images/1643514765/7768/css_is_awseome.png?1643514765" />

Reply:
Sometimes you just gotta love CSS 😂

Reply:
Came here to say I hate CSS.